features/karaf/esh-core/src/main/feature/feature.xml

```diff
@@ -158,6 +158,11 @@
     <bundle>mvn:org.eclipse.smarthome.auth/org.eclipse.smarthome.auth.jaas/${project.version}</bundle>
   </feature>
 
+  <feature name="esh-auth-oauth2" version="${project.version}">
```

Comment:
`esh-auth-oauth2client` might be better, so that it is clear that this is not about server-side auth (as the jaas feature above).

Reply:
Changed to `esh-auth-oauth2client`